vmdb/app/controllers/api_controller/vms.rb

```diff
@@ -85,5 +96,13 @@ def suspend_vm(vm, klass)
     rescue => err
       action_result(false, err.to_s)
     end
+
+    def destroy_vm(vm)
+      desc = "#{vm_ident(vm)} deleting"
+      task_id = queue_object_action(vm, desc, :method_name => "destroy")
```

Comment:
Hi Jason,yes, that was intentional, wanted a single method for setting/queuing the methods for the rest api.  Didn't see any extra logic in the individual _queue methods of benefit.  We can always update the one queue_object_action method to do as you said if needed. Thanks.